src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int maxValues = 100;
```

Comment:
```suggestion
    private static final int MAX_VALUE = 100;
```

Reply:
I tried to make static but maven test failed